### DISCUSSION TITLE
Working with IFG

### DISCUSSION THREAD

**Discussion by aikios** [rdkit/rdkit#3893] [Q&A] [answered]
Hello,

How do I "import" ifg to a ipython notebook (i'm using colab) to work with? I have tried importing from rdkit, but with no luck

**Comment by greglandrum** [comment, marked as answer]
There's information on how to use things from the Contrib folder here:
https://github.com/rdkit/rdkit/issues/2279

**Reply by aikios** [reply]
Thank you!
